### PR TITLE
Copy CDN css and js to local resources

### PR DESCRIPTION
**Pull request by Varunram**
Intended as a fix for #1949

### CODE REVIEW

**Comment by felixmulder** [comment]
Forgot to add the css/js files themselves to the PR? :)

**Comment by felixmulder** [comment]
There are tests for the site structure in `SiteTests.scala`, it would be great if we could make sure these files that you've added locally are indeed copied to the generated sources 👍

**Comment by Varunram** [comment]
> Forgot to add the css/js files themselves to the PR? :)
Geez! Will do, along with the tests

**Comment by Varunram** [comment]
Done!

File: doc-tool/resources/_layouts/main.html [hunk 1]
```diff
@@ -42,21 +40,9 @@
...
-            crossorigin="anonymous"
-        ></script>
-        <script
-            src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.5/js/bootstrap.min.js"
-            integrity="sha384-BLiI7JTZm+JWlgKa0M0kGRpJbF2J8q+qreVrKBC47e3K6BW78kGLrCkeRX6I9RoK"
-            crossorigin="anonymous"
-        ></script>
+        <script src="{{ site.baseurl }}/js/jquery.min.js"></script>
+        <script src="{{ site.baseurl }}/js/tether.min.js"></script>
+        <script src="{{ site.baseurl }}/js/bootstrap.pack.js"></script>
```
Comment (felixmulder) [inline doc-tool/resources/_layouts/main.html]: You're mentioning `bootstrap.pack.js`, but you've added `bootstrap.min.js` as a file.  Highlight js is entirely missing

Comment (Varunram) [inline doc-tool/resources/_layouts/main.html]: Extremely sorry, noticed it just now. I've pushed an update.

**Comment by felixmulder** [comment]
Thanks @Varunram! :tada:

**Comment by Varunram** [comment]
Thanks for your patience!

**Pull request merged**